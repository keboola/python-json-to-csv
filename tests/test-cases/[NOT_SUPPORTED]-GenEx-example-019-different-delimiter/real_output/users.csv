id,name,primary.address_street,primary.address_city,primary.address_state,secondary.address_street,secondary.address_city,secondary.address_state
123,John Doe,Elm Street,New York,,Cemetery Ridge,New York,
234,Jane Doe, Blossom Avenue,,U.K.,1313 Webfoot Walk,Duckburg,Calisota
