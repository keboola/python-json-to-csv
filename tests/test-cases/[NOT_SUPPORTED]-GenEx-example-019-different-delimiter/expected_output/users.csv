"id","name","primary_address_street","primary_address_city","primary_address_state","secondary_address"
"123","John Doe","Elm Street","New York","","{""street"":""Cemetery Ridge"",""city"":""New York""}"
"234","Jane Doe"," Blossom Avenue","","U.K.","{""street"":""1313 Webfoot Walk"",""city"":""Duckburg"",""state"":""Calisota""}"
